src/content/blog/2025_ucc_unitaryhack.md

```diff
@@ -0,0 +1,136 @@
+
+#### Verifying circuit properties for benchmarking
+[GH issue](https://github.com/unitaryfoundation/ucc-bench/issues/27)  
+Bounty: $75  
+Closed by: [WingCode](https://github.com/WingCode)  
+
+**I was drawn in by** the compelling opportunity to enhance benchmarking reliability through direct gate‑set verification. Because the issue was clearly described, I was able to proceed smoothly, encountering minimal difficulty. As a result, I enabled robust checks for gate‑set‑specific compilations, strengthening benchmarking accuracy within UCC.
+
+
+<div style="display: flex; flex-direction: column; align-items: center;">
+    <img style="margin-top: 0; margin-bottom: 0;" src="/images/2025_ucc_unitaryhack/gate-control.png" alt="cartoon of a Matrix Product State (MPS) drawn as a strip of paper with scissors poised to cut at regular intervals" width="200"/>
+    <span style="font-size: 0.9em; color: #666; margin-top: 4px; display: block; text-align: center;">
+        Source: <a href="https://pennylane.ai/qml/demos/tutorial_optimal_control" target="_blank" style="color: #666; text-decoration: underline;">PennyLane gate compilation tutorial</a>
+    </span>
+</div>
+
+**In the future**, I am interested in researching additional transpilation passes within the UCC framework, particularly by leveraging large language models (LLMs). Additionally, I aim to design scalable and predictive models capable of evaluating the logical equivalence of black-box quantum circuits, focusing on metrics such as compute time and memory usage.  
+
+--
+#### Introducing an approximate compilation module
+[GH issue](https://github.com/unitaryfoundation/ucc/issues/381)  
+Bounty: $500  
+Closed by: [ACE07-Sev](https://github.com/ACE07-Sev)  
+
+**Greetings! I am Amir Ali Malekani Nezhad**, a Quantum Compilation Researcher and Developer currently pursuing MPS/MPO utilization for approximate compilation of arbitrary statevectors and operators in O(N) depth.
+
+**What drew me to this issue or to UCC?**
+During UnitaryHack 2025, I had the pleasure of contributing to UCC, a Quantum Compilation library maintained by [Jordan Sullivan](https://github.com/jordandsullivan) and [Brad Chase](https://github.com/bachase). Given my current obsession with anything related to Quantum Compilation, the issue immediately intrigued me. Upon reading about the premise of the issue, and the creative freedom provided by UCC’s supportive team, I proposed an approximate quantum compilation module. This was intended to complement the current compilation pipeline present within UCC to tradeoff slight infidelity with exponential depth reduction for certain class of states known as area-law entangled states.
```

Comment:
Is the area/volume-law entangled states drawing from https://arxiv.org/abs/0808.3773? A reference here would be appreciated (for me!).